script/Ciel/Day 10/10_01_CIEL10_8.txt

```diff
@@ -317,20 +354,23 @@ Enough... I just need to sleep.
 [sha:8253621bd0386647a4f138cc66fdab311682cf8c]{
 -- Page 609, Offset 28776.
 -- 明日になればきっと諦めがついてくれる。
-Come tomorrow, I'm sure I'll have given up.
+I'm sure that by tomorrow, I'll have given up.
 }
 [sha:752e01de1e0f8c0919c233f7ff0773816a4fe87c]{
 -- Page 610, Offset 28777.
 -- いつも通り。戻れる保証のない、あの暗い眠りに落ちれば、何もかも削ぎ落とされて新しい朝を迎えられる。
-If I let myself fall like always into that dark sleep from which I have no guarantee to ever wake, everything will be worn away and I'll be able to greet the new morning with open arms.
+I'll submit myself once again to sleep,
+ knowing as always there's no guarantee I'll ever wake,
+ so that all these feelings can be scraped away, and I can greet the morning with open arms.
```

Comment:
feels like

once again submit myself to sleep > submit myself once again to sleep

but it doesn't quite matter